backend/lcfs/web/api/initiative_agreement/services.py

```diff
@@ -208,16 +209,27 @@ async def director_approve_initiative_agreement(
             initiative_agreement.transaction_effective_date = datetime.now().date()
 
         await self.repo.refresh_initiative_agreement(initiative_agreement)
-        await self._perform_notificaiton_call(initiative_agreement)
 
-    async def _perform_notificaiton_call(self, ia, re_recommended=False):
+    async def _perform_notificaiton_call(self, ia, returned=False):
```

Comment:
suggestion: Fix spelling of notification

Reply:
fixed